### PR TITLE
doc: portuguese documentation translation

### PR DESCRIPTION
**Pull request by renesoaresse**
Translation of all documentation to Brazilian Portuguese.

### CODE REVIEW

File: docs/Custom-iOS.portuguese.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+// RCTCustomWebView.m
+@interface RCTCustomWebView ()
+
+@property (nonatomic, copy) RCTDirectEventBlock onNavigationCompleted;
+
+@end
+```
+
+Then expose it in the view manager's `@implementation`.
```
Comment (TheAlmightyBob) [inline docs/Custom-iOS.portuguese.md:84]: Was this intentionally left in English?

File: docs/Reference.portuguese.md [hunk 2]
```diff
@@ -0,0 +1,1639 @@
...
+
+### `injectedJavaScriptForMainFrameOnly`[⬆](#props-index)<!-- Link gerado com jump2header -->
+
+If `true` (default; mandatory for Android), loads the `injectedJavaScript` only into the main frame.
+
+If `false`, (only supported on iOS and macOS), loads it into all frames (e.g. iframes).
+
+| Tipo | Requerido | Plataforma                                          |
+| ---- | --------- | ------------------------------------------------- |
+| bool | Não        | iOS and macOS (only `true` supported for Android) |
```
Comment (TheAlmightyBob) [inline docs/Reference.portuguese.md:229]: Is this intentionally left in English?

**Review by TheAlmightyBob — COMMENTED** [review]
Wow. Are you just trying to create more work for @InTheNooB? 😉

File: docs/Guide.portuguese.md [hunk 3]
```diff
@@ -0,0 +1,552 @@
...
+
+##### iOS
+
+O iOS, você terá que fornecer seu próprio código para baixar os arquivos. Você pode fornecer um retorno de chamada `onFileDownload` para o componente WebView como um prop. Se o RNCWebView determinar que um download de arquivo precisa ocorrer, a URL onde você pode baixar o arquivo será dado a `onFileDownload`. A partir desse retorno de chamada, você pode baixar esse arquivo da maneira que desejar.
+
+NOTA: O iOS 13+ é necessário para a melhor experiência de download possível. No iOS 13, a Apple adicionou uma API para acessar cabeçalhos de resposta HTTP, que
+é usado para determinar se uma resposta HTTP deve ser um download. No iOS 12 ou anterior, apenas os tipos MIME que não podem ser renderizados pela webview serão
+acionar chamadas para `onFileDownload`.
+
+Example:
```
Comment (TheAlmightyBob) [inline docs/Guide.portuguese.md:240]: Should this be "Exemplo"?

**Comment by renesoaresse** [comment]
> Wow. Are you just trying to create more work for @InTheNooB?

kkkkk nothing, just contributing a little to the RN community. @TheAlmightyBob

File: docs/Custom-iOS.portuguese.md (hunk 1 above)
Comment (renesoaresse) [reply docs/Custom-iOS.portuguese.md:84]: No, it passed the review

File: docs/Reference.portuguese.md (hunk 2 above)
Comment (renesoaresse) [reply docs/Reference.portuguese.md:229]: No, it passed the review

File: docs/Guide.portuguese.md (hunk 3 above)
Comment (renesoaresse) [reply docs/Guide.portuguese.md:240]: No, it passed the review

File: docs/Custom-iOS.portuguese.md (hunk 1 above)
Comment (renesoaresse) [reply docs/Custom-iOS.portuguese.md]: Made the correction!

File: docs/Reference.portuguese.md (hunk 2 above)
Comment (renesoaresse) [reply docs/Reference.portuguese.md]: Made the correction!

File: docs/Guide.portuguese.md (hunk 3 above)
Comment (renesoaresse) [reply docs/Guide.portuguese.md]: Made the correction!

**Review by TheAlmightyBob — APPROVED** [review]

**Comment by TheAlmightyBob** [comment]
Obrigado @renesoaresse!

**Pull request merged**